mlflow/tracking/__init__.py

```diff
@@ -225,6 +223,13 @@ def _get_unix_timestamp():
     return int(time.time() * 1000)
 
 
+def _get_artifact_repo(run_info, store):
```

Comment:
I wonder if we can just remove the handling of legacy `run_info`s (we only need to do that for runs created before we started storing artifact URIs in runs) & it seems it'd simplify our logic in a number of places

Reply:
I think we probably should do that in a different PR to simplify this one.